javascript/javascript3/week2/fetching-exercise/Fetching_waiting.js

```diff
@@ -0,0 +1,33 @@
+console.log("script loaded");
+
+new Promise(function(resolve, reject) {
+    setTimeout(() => {
+        resolve("Whoops!");
+    }, 3000);
+})
+
+fetch("http://api.open-notify.org/astros.json")
+    // .then below runs when the remote server responds
+    .then(function(response) {
+        // response.text() returns a new promise that resolves with the full response text
+        // when it loads
+        return response.text();
```

Comment:
Here you are converting to a string and not an object. Was this to alert it?
You can check the content type first if you don't know
https://stackoverflow.com/questions/37121301/how-to-check-if-the-response-of-a-fetch-is-a-json-object-in-javascript